### PR TITLE
Change hammock to build with skipper

### PR DESCRIPTION
**Pull request by ravid-stratoscale**
@shay-stratoscale @eran-stratoscale @adir-stratoscale @rom-stratoscale @eyal-stratoscale @shimshon-stratoscale @mickey-stratoscale

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,31 +2,17 @@
...
 
 import os
-import subprocess
 import setuptools
 
 
-PKG_INFO = 'PKG-INFO'
-
-
-def version():
```
Comment (eran-stratoscale) [inline setup.py]: Since your build container is python base you can't use git_version, so add it back.

File: Makefile [hunk 2]
```diff
@@ -7,13 +7,6 @@ TEST ?= --config setup.cfg
 TOXENV ?= py27
 ENV := TOXENV=$(TOXENV)
```
Comment (shay-stratoscale) [inline Makefile]: do you still need this?

File: Dockerfile.build [hunk 3]
```diff
@@ -0,0 +1,7 @@
+FROM python:2
```
Comment (adir-stratoscale) [inline Dockerfile.build]: ummm... after looking at the tox.ini, i saw that hammock support also python 3. 
using this build container we don't check that python 3 support wasn't break.

Comment (adir-stratoscale) [inline Dockerfile.build]: maybe we should have another build container for python 3. the skipper.yaml will point to the python 2 build container by default, and if someone wants to test python 3 support he will have to change the build container in command-line.

Comment (ravid-stratoscale) [inline Dockerfile.build]: the pylint is checking also for python 3

Comment (adir-stratoscale) [inline Dockerfile.build]: but what about the unittests?

**Pull request merged**